index.js

```diff
@@ -378,11 +398,13 @@ class PluginXrpPaychan extends PluginBtp {
       util.fundChannel({
         api: this._api,
         channel: this._outgoingChannel,
-        amount: util.xrpToDrops(this.baseToXrp(this._channelAmount)),
+        amount: util.xrpToDrops(this._outgoingChannelDetails.amount),
```

Comment:
What does this indicate?

Reply:
Rather than increasing by the initial channel capacity, this increases by the current channel capacity. That means there will be fewer fund transactions, because it will double each time you reach half the capacity